libraries/AP_HAL_ChibiOS/hwdef/uav-dev-fc-um982/README.md

```diff
@@ -0,0 +1,149 @@
+- **microSD**
+- **DEBUG**
+  - Tag-Connect
+
+
+## Pinout
+![top](uav-dev-fc-um982.png)
+
+#### S1 - SERIAL4 & SERIAL5 6 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | USART6_TX       | 3.3V         |
+| 3   | USART6_RX / RCIN| 3.3V         |
+| 4   | UART7_TX        | 3.3V         |
+| 5   | UART7_RX        | 3.3V         |
+| 6   | GND             | GND          |
+
+#### PWM - PWM / DShot / Telemetry - 8 Pin JST-GH
+| Pin | Signal Name           | Voltage      |
+|-----|-----------------------|--------------|
+| 1   | PWM / DShot 1 BIDIR   | 3.3V         |
+| 2   | PWM / DShot 2         | 3.3V         |
+| 3   | PWM / DShot 3 BIDIR   | 3.3V         |
+| 4   | PWM / DShot 4         | 3.3V         |
+| 5   | PWM / DShot 5         | 3.3V         |
+| 6   | PWM / DShot 6         | 3.3V         |
+| 7   | USART3_RX / TELEM     | 3.3V         |
+| 8   | GND                   | GND          |
+
+#### C1 - CAN FD 1 - 4 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | CAN_H           |              |
+| 3   | CAN_L           |              |
+| 4   | GND             | GND          |
+
+#### C2 - CAN FD 2 - 4 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | CAN_H           |              |
+| 3   | CAN_L           |              |
+| 4   | GND             | GND          |
+
+#### Debug - 6 Pin Tag-Connect
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +3.3V           | 3.3V         |
+| 2   | SWDIO           | 3.3V         |
+| 3   | NRST            | 3.3V         |
+| 4   | SWCLK           | 3.3V         |
+| 5   | GND             | GND          |
+| 6   | not connected   |              |
+
+#### S2 - SERIAL1 & GNSS - 6 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | USART1_TX       | 3.3V         |
+| 3   | USART1_RX       | 3.3V         |
+| 4   | GNSS_PPS        | 3.3V         |
+| 5   | GNSS_EVENT      | 3.3V         |
+| 6   | GND             | GND          |
+
+#### ETH - Ethernet - 4 Pin JST-GH
+| Pin | Signal Name       |
+|-----|-------------------|
+| 1   | RX-               |
+| 2   | RX+               |
+| 3   | TX-               |
+| 4   | TX+               |
+
+## UART Mapping
+
+|Name|Function|
+|:-|:-|
+|SERIAL0|USB|
+|SERIAL1|USART1 (MAVLink)|
+|SERIAL2|UART8 (GPS1) GNSS UM982|
+|SERIAL3|USART2 (Scripting) GNSS UM9892|
+|SERIAL4|USART6 (RCIN)|
+|SERIAL5|UART7 (MAVLink)|
+|SERIAL6|USART3 (ESC telemetry)|
+|SERIAL7|OTG2 (SLCAN)|
+
+Any UART can be re-tasked by changing its protocol parameter.
+
+## RC Input
+
+RC input is configured on S1 / SERIAL4 / UART6. It supports all RC protocols.
```

Comment:
once I understand the RC in situation, I will have verbiage for this section

Reply:
Now only UART6 for RCIN